pkg/cmd/util/clientcmd/factory.go

```diff
@@ -201,7 +201,11 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if !ok {
 				return nil, errors.New("provided options object is not a BuildLogOptions")
 			}
-			buildsForBCSelector := labels.SelectorFromSet(map[string]string{buildapi.DeprecatedBuildConfigLabel: t.Name})
+			buildConfigLabels := labels.Set{
+				buildapi.BuildConfigLabel: t.Name,
```

Comment:
at what point will we be able to use the new label in selectors? we need to track when these new labels are added, and when we plan to remove support for the old one. I would expect a comment here, and a tracking document